### ISSUE TITLE
Improve the content in the About page

### ISSUE THREAD

**Issue by Abhay5855** [jeoffreyduke/clann#15]
![195604377-7d58b201-b12a-41bf-919e-2507a9f3bf99](https://user-images.githubusercontent.com/71277980/195682378-b94d5437-540a-42d5-a471-3e9f2373d0b7.png)

**Comment by jeoffreyduke** [comment]
Any update?

**Comment by Abhay5855** [comment]
@jeoffreyduke Yes started working on it will give any update till tomorrow noon

**Comment by jeoffreyduke** [comment]
> @jeoffreyduke Yes started working on it will give any update till tomorrow noon

Are you done?

**Comment by khushibansal13** [comment]
can i work on this?
